lib/Doctrine/DBAL/Connection.php

```diff
@@ -96,6 +96,13 @@ class Connection implements DriverConnection
     protected $_conn;
 
     /**
+     * The connection name.
+     *
+     * @var string
+     */
+    protected $_name;
```

Comment:
I need to get the connection name to retrieve the related shard manager in https://github.com/doctrine/DoctrineMigrationsBundle/pull/132